README.rst

```diff
@@ -20,8 +20,13 @@ Silicone
 .. sec-begin-long-description
 .. sec-begin-index
 
-Silicone is a Python package which can be used to automatically fill in detail in emissions scenarios.
-For example, adding emissions of aerosol precurors and methane based on carbon dioxide and nitrous oxide emissions pathways or adding regional emissions based on a global emissions pathway.
+Silicone is a Python package which can be used to infer predictions about certain types
+of emissions from other predicted emissions. It is intended to ensure more integrated
+assessment models (IAMs) have predictions of all climate-relevant emissions.
```

Comment:
```suggestion
assessment model (IAM) data so that their scenarios quantify more climate-relevant emissions than is natively reported by the IAMs themselves.
```